packages/forms/src/UIForm/fieldsets/Fieldset/Fieldset.component.test.js

```diff
@@ -29,6 +29,44 @@ describe('Fieldset widget', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	it('should not render fieldset legend without any title', () => {
+		// given
+		const schema = {
+			items: [
+				{
+					key: ['user', 'firstname'],
+					type: 'text',
+					schema: { type: 'string' },
+				},
+				{
+					key: ['user', 'lastname'],
+					type: 'text',
+					schema: { type: 'string' },
+				},
+			],
+		};
+
+		// when
+		const wrapper = shallow(<Fieldset schema={schema} />);
+
+		// then
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
please don t use snapshot that way.
you should find(legend) and assert there is no.